packages/@azure/servicebus/data-plane/package.json

```diff
@@ -22,18 +22,17 @@
     "node": ">=6.0.0"
   },
   "dependencies": {
-    "@azure/amqp-common": "^1.0.0-preview.1",
+    "@azure/amqp-common": "git+https://github.com/Azure/amqp-common-js.git#master",
+    "@azure/ms-rest-nodeauth": "^0.9.1",
```

Comment:
Let's make this ^0.9.2 to ensure we have the type fix to https://github.com/Azure/ms-rest-nodeauth/issues/49